### DISCUSSION TITLE
Posts not appearing after hitting the "post" button

### DISCUSSION THREAD

**Discussion by docpop** [pixelfed/pixelfed-rn#120] [Bug Reports]
After writing a post with a few images, I hit the post button, and I'm taken back to the main timeline in my app. It appears as if everything worked, but the post never appears in my feed. This is extra annoying because it means I have to re-write all of the alt text and try again. I've had something similar happen where I hit post and was  shown a loading wheel that didn't go away, but I've also had it happen where there was no loading wheel. Hitting the post button took me straight to the home feed. When tapping the plus button, I see an empty post waiting to be filled out. It woudl be nice if Pixelfed could keep track of when a post fails, so it can store it in an draft or on that "+" button so we can try again. 
I'm using the current Pixelfed version (as of this morning) on iOS 18.0 for iPhone XR. This issue doesn't happen all the time. I just tried it and the post worked as usual, so I can't recreate it consistently.

**Comment by bucknam** [comment]
I’ve had this problem repeatedly today (posting to my account on pixelfed.social via the iOS app). It’s frustrating because I put a lot of effort into writing the caption and the alt text and it just disappears.

**Comment by DarKnightofCydonia** [comment]
I figured it out, at least for Android - it won't accept heic files (high compression), only jpeg. This is a pretty big issue though since a lot of people save their images in these high efficiency formats nowadays.

**Comment by docpop** [comment]
Good catch! I have that issue with some other web-based platforms (like
WordPress). Hopefully there's an easy fix for it.

On Thu, Jan 16, 2025 at 2:54 AM DarKnightofCydonia ***@***.***>
wrote:

> I figured it out, at least for Android - it won't accept heic files (high
> compression), only jpeg. This is a pretty big issue though since a lot of
> people save their images in these high efficiency formats nowadays.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pixelfed/pixelfed-rn/discussions/120#discussioncomment-11853470>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABJT3GADOREC7ZWIHI5DNHD2K6FWVAVCNFSM6AAAAABNB5UKG2VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCOBVGM2DOMA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 


Doctor M. Popular

415-724-9267

www.docpop.org
